src/plugins/vis_augmenter/public/utils/utils.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { get } from 'lodash';
+import { Vis } from '../../../visualizations/public';
```

Comment:
nit - for consistency

```suggestion
import { Vis } from '../../../../plugins/visualizations/public';
```